.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint
```

Comment:
Hi 👋, @r3nanp we have one build where we run the tests and build the application, how about instead of creating a new workflow integrate the linter in the existing one? I guess this would be more simple,

Reply:
Hey @vitormalencar , sure.